### ISSUE TITLE
A Question-Focused Multi-Factor Attention Network for Question Answering

### ISSUE THREAD

**Issue by Toshiki-Sasaki** [Toshiki-Sasaki/papers#61]
https://arxiv.org/abs/1801.08290
Souvik Kundu
NAACL 2018

## Abstract
QAタスクを解くためのneural networkモデルが最近提案されている．
それは主にpassage-questionの関係を捉えることに焦点を置いている．
しかしながら，それらは複数の文を用いた推論や共参照などのような深い理解を達成するため極めて重要な，複数の文をまたぐ関係のある文をつなぐための最小の能力しか持っていない．
それらはQAでとても重要な役割を果たす質問と答えの型に焦点を置いていない．
この論文では，我々は答えの抽出のために新しいend-to-end question-focused multi-factor attention networkを提案します．
tensor-based transformation を用いた Multi-factor attentive encoding は意味のある事実を，複数の文に位置していても集めます．
答えの型をそれとなく推論するために，我々は質問の中の重要な単語に基づいたquestion vectorをエンコードするためのmax-attentional question aggregationメカニズムを提案します．
予測時は，我々は質問文のwh-wordのsequence-level encodingと，質問の型に関する情報の追加のソースとしてそれに続く単語を組み込みます．
我々の提案したモデルは，大きいデータサイズのQAデータセットであるNewsQA, TriviaQA, SearchQAでSOTAを達成しました．

## ポイント
タスク：抽出QA
データセット：NewsQA, TriviaQA, SearchQA
特徴1：multi-fact attentionの実現のためにtensor-based transformationを用いる．(bilinearのtensor版)
特徴2：質問と答えの型を質問に注目したattentionにより求める．
### 実験
word embedding : GloVe 300-dim (no updating during training)
OOV : zero-vector
Charcter-level embedding : 50-dim
hidden-dim of LSTM : 150
dropout : 0.3 in every learnable layer
推論に用いる文の数 : 4 (固定)
mini-batch : 60
Optimizer : Adam (learning rate : 0.001)
clipnorm(L2正則化の最大勾配値) :  5
### 結果
multi-fact attention は2pointほど
question-dependent passageが15pointほどモデルのパフォーマンスに関係．
